website/docs/r/nsxt_edgegateway.html.markdown

```diff
@@ -351,6 +351,10 @@ can be used to lookup ID by name.
 * `external_network` - (Optional, *VCD 10.4.1+*, *v3.11+*) attaches NSX-T Segment backed External
   Networks with a given [configuration block](#edgegateway-subnet-external-network). It *does not
   support IP Spaces*.
+* `read_limit_unused_ip_count` - (Optional, *v3.13+*) Sets a limit of IPs to count for
+  `used_ip_count` and `unused_ip_count` attributes to avoid exhausting compute resource while
+  counting IPs in large IPv6 subnets. Defaults to `1000000`, update is a no-op, but will affect
+  newly read data.
```

Comment:
Though description is already very helpful, I think implications of using this field are still unclear. Maybe add a sentence (or example separately somewhere below) on the actual drawback of using it?

Reply:
Please check once more as I have addressed other comments and hopefully improved this as well.
The point is - nothing in operations except values of `used_ip_count` and `unused_ip_count` are affected.